language/move-prover/bytecode/src/verification_analysis_v2.rs

```diff
@@ -270,6 +300,7 @@ fn compute_funs_that_modify_inv(
                 let fun_target = targets.get_target(&fun_env, &variant);
                 let modified_memory = usage_analysis::get_modified_memory_inst(&fun_target);
                 // Add functions to set if it modifies mem used in invariant
+                // TODO: This should be using unification.
```

Comment:
I can follow-up on this

Reply:
There are pervasive problems like this.  I suggest you wait until we have a chance to discuss how to reorganize the code so we can fix them all.

Reply:
Will do